src/core_plugins/console/public/src/input.js

```diff
@@ -178,8 +178,13 @@ export function initializeInput($el, $actionsEl, $copyAsCurlEl, output) {
 
             var warnings = xhr.getResponseHeader("warning");
             if (warnings) {
-              warnings = _.map(warnings.split(", "), function (warning) {
-                  return "#! Deprecation: " + warning;
+              // pattern for valid warning header
+              var re = /\d{3} [^ ]+ \"([^"]*)\"( \"[^"]*\")/
```

Comment:
Bah - ignore me. This would work (which is what I think you were suggesting):

    "((?:\\"|[^"])*)"

Which says:

* match `*`
* capture:
  * zero or more times: match `\"` or any character except `"`
* match `"`